.github/actions/deploy/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:10.16.3-jessie
+
+RUN dpkg --add-architecture i386
+RUN apt update
+RUN apt -y install python make libxkbfile-dev libxkbfile-dev:i386 libx11-dev libx11-dev:i386 libxss-dev gcc-multilib g++-multilib rpm
```

Comment:
fun fact, not the biggest deal here, but each of these lines creates a new "layer" in the Docker image and can carry a reasonable performance hit. since it's an automated build it won't matter too much but it's common to join these into one `RUN` statement. put differently, we've created these on separate `RUN` lines for organizational purposes but that creates an unintended side-effect in the Docker image we didn't want.

line-continuation marks and indentation are our friends, and now we can alphabetize the dependencies and make future diffs clearer

```Dockerfile
RUN dpkg --add-architecture i386 \
    && apt update \
    && apt -y install \
         g++-multilib \
         gcc-multilib \
         libx11-dev libx11-dev:i386 \
         libxkbfile-dev libxkbfile-dev:i386 \
         libxss-dev \
         make \
         python \
         rpm
```